src/main/resources/reference.conf

```diff
@@ -422,67 +425,76 @@ resourceTypes = {
       update_study = {
         description = "Change study metadata"
       }
-      delete-study = {
+      delete = {
         description = "Delete study, including data"
       }
-      edit-study = {
+      edit_study = {
         description = "Change study name and description"
       }
-      ingest-data = {
+      ingest_data = {
         description = "Add data to a study"
       }
-      update-data = {
+      update_data = {
         description = "Change or delete data of a study"
       }
-      create-dataset = {
+      create_dataset = {
         description = "Create a dataset that includes this study"
       }
+      share_policy::custodian = {
+        description = "Can grant and revoke a users' custodian permission"
+      }
     }
     ownerRoleName = "steward"
     roles = {
       steward = {
-        roleActions = ["update_study",  "delete-study",  "edit-study",  "ingest-data",  "update-data",  "create-dataset",  "share_policy::custodian",  "delete"]
+        roleActions = ["update_study",  "delete",  "edit_study",  "ingest_data",  "update_data",  "create_dataset",  "share_policy::custodian"]
       }
       custodian = {
-        roleActions = ["create-dataset"]
+        roleActions = ["create_dataset"]
       }
       ingester = {
-        roleActions = ["ingest-data"]
+        roleActions = ["ingest_data"]
       }
     }
     reuseIds = true
   }
   dataset = {
     actionPatternObjects = {
-      update-dataset = {
+      update_dataset = {
         description = "Update dataset"
       }
-      delete-dataset = {
+      delete = {
         description = "Delete dataset"
       }
-      edit-dataset = {
+      edit_dataset = {
         description = "Change dataset name and description"
       }
-      read-data = {
+      read_data = {
         description = "Read data from the dataset"
       }
-      discover-data = {
+      discover_data = {
```

Comment:
Do you mind updating these in the[ IAM doc](https://docs.google.com/document/d/1iL_zwDiaXYEMYsWfgK6dCftOwvaY8AcxjVKFXcEMjS8/edit#heading=h.eklp7mvtgs15)?

Reply:
doc has been updated to match the config file